editoast/src/main.rs

```diff
@@ -86,6 +86,18 @@ async fn runserver(
     // Setup shared states
     let infra_caches = Data::new(CHashMap::<i64, InfraCache>::default());
 
+    // Setup sentry
+    if args.sentry_dsn.is_some() && args.sentry_env.is_some() {
```

Comment:
can you pattern-match instead, and display a warning if either is set but not both?
this way, if someone sets sentry_dsn but not sentry_env, if would immediately be obvious that something is off